### PR TITLE
handle error cannot open file attachment

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/44899

## Description
added ignoring for `ErrorCannotOpenFileAttachment` response code.

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
 - [x] No

## Must have
- [ ] Tests

### CODE REVIEW

File: Packs/EWS/Integrations/EWSO365/EWSO365.py [hunk 1]
```diff
@@ -2033,82 +2098,20 @@ def parse_incident_from_item(item):
...
+                    {
+                        "type": label_attachment_type,
+                        "value": get_attachment_name(attachment.name),
+                    }
+                )
+                labels.append(
+                    {"type": label_attachment_id_type, "value": attachment.attachment_id.id}
+                )
+            except ErrorCannotOpenFileAttachment:
+                demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSO365/EWSO365.py:2114]:
```suggestion
                demisto.error("Could not open attachment file for message_id: {}".format(item.message_id))
```

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 2]
```diff
@@ -1103,87 +1104,95 @@ def parse_incident_from_item(item, is_fetch):
...
-            incident['attachment'] = []
-            for attachment in item.attachments:
-                if attachment is not None:
-                    attachment.parent_item = item
-                    file_result = None
-                    label_attachment_type = None
-                    label_attachment_id_type = None
-                    if isinstance(attachment, FileAttachment):
+        try:
+            demisto.info('ErrorCannotOpenFileAttachment first')
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:1108]: ?

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 3]
```diff
@@ -1103,87 +1104,95 @@ def parse_incident_from_item(item, is_fetch):
...
+            if item.attachments:
+                incident['attachment'] = []
+                for attachment in item.attachments:
+                    if attachment is not None:
                         try:
-                            if attachment.content:
-                                # file attachment
-                                label_attachment_type = 'attachments'
-                                label_attachment_id_type = 'attachmentId'
+                            demisto.info('ErrorCannotOpenFileAttachment second')
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:1114]: ?

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 4]
```diff
@@ -1103,87 +1104,95 @@ def parse_incident_from_item(item, is_fetch):
...
+                                    incident['attachment'].append({
+                                        'name': get_attachment_name(attachment.name) + ".eml"
+                                    })
+
+                            labels.append({'type': label_attachment_type, 'value': get_attachment_name(attachment.name)})
+                            labels.append({'type': label_attachment_id_type, 'value': attachment.attachment_id.id})
+                        except ErrorCannotOpenFileAttachment:
+                            demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
+        except ErrorCannotOpenFileAttachment:
+            demisto.error("Could not open file attachment file for message_id: {}".format(item.message_id))
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:1195]: can we be minimize the scope of the try-except?
relevant for both integrations
```suggestion
            demisto.error("Could not open attachment file for message_id: {}".format(item.message_id))
```

File: Packs/EWS/Integrations/EWSO365/EWSO365.py (hunk 1 above)
Comment (anara123) [reply Packs/EWS/Integrations/EWSO365/EWSO365.py:2114]: See my comment in the issue.
If we proceed in this direction with exception, we should catch the exception on single attachment level.
If there are 5 attachments in the email, only one of them should be dropped

**Pull request closed without merging**